sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobServiceImpl.java

```diff
@@ -1512,6 +1621,93 @@ public void onFailure(@NonNull Call<ResponseBody> call, @NonNull Throwable t) {
         }
     }
 
+    private ContainerDeleteResponse deleteContainerWithRestResponseIntern(String containerName,
+                                                                          Integer timeout,
+                                                                          String version,
+                                                                          String leaseId,
+                                                                          OffsetDateTime ifModifiedSince,
+                                                                          OffsetDateTime ifUnmodifiedSince,
```

Comment:
In java we also use BlobRequestConditions here. Not super sure how I feel about that, but there it is.

Reply:
I can pass in BlobRequestConditions. Its a little weird though just cause 2 params dont apply here though

Reply:
updated to BlobRequestConditions